_events/sr2019/competition.md

```diff
@@ -18,11 +18,9 @@ On Sunday morning, the league matches will continue. Once these have concluded,
 
 ## Tickets and Media Consent
 
-Competitors need to bring along a ticket to be allowed entry. Anyone under the
-age of 18 must be accompanied by an adult. Student Robotics will announce when
-tickets are available.
+Competitors need to bring along a ticket to be allowed entry. Anyone under the age of 18 must be accompanied by an adult. To obtain a ticket, competitors need to sign a [media consent form][mcf-link] and [send back to us][teams-contact]. Those under the age of 16 will need to get a parent or guardian to sign on their behalf. Once processed, you'll receive an automated email with a link to your ticket.
```

Comment:
Hrm, we should probably (somewhere) specify what the "event" we want them to put on the form is.